spykes/datasets.py

```diff
@@ -29,25 +30,23 @@ def load_reward_data(dpath='~/spykes_data/reward/'):
     sess_four_mat: .mat file
         Monkey M, Session 4
     """
-
+    dpath = os.path.expanduser(dpath)
     if not os.path.exists(dpath):
         os.makedirs(dpath)
 
     fname = os.path.join(dpath, 'Mihili_07112013.mat')
 
     if not os.path.exists(fname):
 
-        urllib.urlretrieve('https://ndownloader.figshare.com/files/5652051',
-                           fname)
+        _fetch_file('https://ndownloader.figshare.com/files/5652051', fname)
```

Comment:
@choldgraf i'd hate to hold you back from actually using `spykes`. +1 for using `download` as a dependency. thanks!